druid/examples/cursor.rs

```diff
@@ -83,16 +73,52 @@ fn ui_builder() -> impl Widget<AppState> {
         .padding(50.0)
 }
 
+#[derive(Clone, Data, Lens)]
+struct AppState {
+    cursor: Rc<Cursor>,
+    custom: Option<Rc<Cursor>>,
+    // TODO refer to docs once they exist
```

Comment:
We can link to the docs for `Data`, for now?

Reply:
do you know of any way to link to the latest docs instead of 0.6? I feel like linking to 0.6 would just become a forgotten link that will get outdated.

Reply:
I don't think there's a way to do that for docs in examples. :(